@stellar/design-system-website/src/componentPreview/projectLogoPreview.tsx

```diff
@@ -0,0 +1,5 @@
+import { ComponentPreview } from "@site/src/components/PreviewBlock";
+
+export const projectLogoPreview: ComponentPreview = {
+  options: [],
```

Comment:
We don't have any props to play with right now, but we can keep the file for consistency.